src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -23,107 +22,305 @@ internal static class TrustedSignersCommand
+                    {
+                        return await ExecuteCommand(TrustCommand.Repository, algorithm: null, allowUntrustedRootOption.HasValue(), owners: owners, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: package.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("certificate", (certificateCommand) =>
+                {
+                    certificateCommand.Description = Strings.TrustRepositoryCommandDescription;
+
+                    CommandOption algorithm = certificateCommand.Option(
+                        "--algorithm",
+                        Strings.TrustCommandAlgorithm,
+                        CommandOptionType.SingleValue);
+
+                    CommandOption allowUntrustedRootOption = certificateCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = certificateCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    certificateCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption verbosity = certificateCommand.VerbosityOption();
+
+                    CommandArgument name = certificateCommand.Argument("<NAME>",
+                                               Strings.TrustedCertificateSignerNameToAdd);
+                    CommandArgument fingerprint = certificateCommand.Argument("<FINGERPRINT>",
+                                               Strings.TrustCertificateFingerprint);
+
+                    certificateCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Certificate, algorithm, allowUntrustedRootOption.HasValue(), owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: null, fingerprint: fingerprint.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("source", (sourceCommand) =>
+                {
+                    sourceCommand.Description = Strings.TrustSourceCommandDescription;
+
+                    CommandOption configFile = sourceCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    sourceCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption owners = sourceCommand.Option(
+                        "--owners",
+                        Strings.TrustCommandOwners,
+                        CommandOptionType.MultipleValue);
```

Comment:
As per [doc](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-trust#options-5), `--owners` option accepts list of values separated by semi-colon. 

Copied from examples section in the doc, `dotnet nuget trust source NuGetTrust https://api.nuget.org/v3/index.json --owners "Nuget;Microsoft"`

I think `CommandOptionType` for this option should be `SingleValue` instead of `MultipleValue`. This comment applies to all the sub-commands which has `owners` as an option.